crates/wascc-provider/src/lib.rs

```diff
@@ -371,11 +371,12 @@ fn wascc_run(
     }
 
     if actor_caps.contains(&HTTP_CAPABILITY.to_owned()) {
-        env.insert("PORT".to_string(), port_assigned.to_string());
+        let mut httpenv = env.clone();
+        httpenv.insert("PORT".to_string(), port_assigned.to_string());
```

Comment:
My only concern here (and with log path) is that if a user specifies an env variable of `PORT` it will screw up the port configuration

Reply:
Right. I was considering that too. However, the internal port number the http server is listening on is set up internally, so the user would not know what port to specify.

Log path is the same way: these are internal temporary ephemeral paths that the users should not be able to specify. It would be simple for a user to set a LOG_PATH that could read another actor's logs, leaking sensitive information to an attacker.

Reply:
And we are overwriting, so I think it should be ok